TeamCode/src/main/java/com/acmerobotics/relicrecovery/path/LinearPath.java

```diff
@@ -0,0 +1,216 @@
+package com.acmerobotics.relicrecovery.path;
+
+import com.acmerobotics.relicrecovery.localization.Vector2d;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Created by Ryan on 4/5/2017.
+ */
+public class LinearPath {
```

Comment:
This is copied from last year. It was created primarily to facilitate the pure pursuit controller. As a result, there are several methods that aren't needed. We can keep them for now; I just tried to modify the class as little as possible.